code/reposadolib/reposadocommon.py

```diff
@@ -475,6 +476,79 @@ def writeLocalCatalogs(applecatalogpath):
     # now write filtered catalogs (branches) based on this catalog
     writeBranchCatalogs(localcatalogpath)
 
+
+def readXMLfile(filename):
+    '''Return dom from XML file or None'''
+    try:
+        dom = minidom.parse(filename)
+    except ExpatError:
+        print_stderr(
+            'Invalid XML in %s', filename)
+        return None
+    except IOError, err:
+        print_stderr(
+            'Error reading %s: %s', filename, err)
+        return None
+    return dom
+
+
+def writeXMLtoFile(node, path):
+    '''Write XML dom node to file'''
+    xml_string = node.toxml()
+    try:
+        fileobject = open(path, mode='w')
+        print >> fileobject, xml_string
+        fileobject.close()
+    except (OSError, IOError):
+        print_stderr('Couldn\'t write XML to %s' % path)
+
+
+def remove_config_data_attribute(product_list):
+    check_or_remove_config_data_attribute(product_list, remove_attr=True)
+
+
+def check_or_remove_config_data_attribute(product_list, remove_attr=False):
+    '''Remove the type="config-data" attribute from the distribution options for
+    a product. This makes softwareupdate find and display updates like
+    XProtectPlistConfigData and Gatekeeper Configuration Data, which it normally
+    does not.'''
```

Comment:
The doc string needs to be updated to reflect the change in how the function works.